com.unity.formats.fbx/Editor/Sources/Recorders/FbxRecorder/FbxRecorder.cs

```diff
@@ -0,0 +1,83 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEditor.Recorder;
+using UnityEditor.Recorder.Input;
+using UnityEditor;
+
+namespace UnityEditor.Formats.Fbx.Exporter
+{
+    class FbxRecorder : GenericRecorder<FbxRecorderSettings>
+    {
+        protected override void RecordFrame(RecordingSession ctx)
+        {
+
+        }
+
+        protected override void EndRecording(RecordingSession session)
+        {
+            var ars = (FbxRecorderSettings)session.settings;
```

Comment:
why "ars"? Can you rename to "settings"?

Reply:
I think this is 'ars' for 'animation recorders'. I'll change it